clojure/resources/firn/_firn_starter/layouts/default.clj

```diff
@@ -1,8 +1,8 @@
 (defn default
-  [{:keys [render partials build-url]}]
+  [{:keys [render partials build-url site-title site-author site-desc]}]
```

Comment:
```suggestion
  [{:keys [render partials build-url site-title site-author site-desc] :as config}]
```

We should probably just keep the map value around so that we can pass it to the `head` partial`, then update the head partial to deconstruct what it needs from the config map.

Reply:
Good call.